pynobo.py

```diff
@@ -546,8 +564,7 @@ async def async_discover_hubs(serial="", ip=None, autodiscover_wait=3.0, loop=No
         :return: a set of hubs matching that serial, ip address or both
         """
 
-        loop = asyncio.get_event_loop() if loop is None else loop
-        transport, protocol = await loop.create_datagram_endpoint(
+        transport, protocol = await asyncio.get_running_loop().create_datagram_endpoint(
             lambda: nobo.DiscoveryProtocol(serial, ip),
             local_addr=('0.0.0.0', 10000),
             reuse_port=True)
```

Comment:
I know this is not from this review, but can we drop this parameter. It breaks compatibility with "Windows and some Unixes":
https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.create_datagram_endpoint